frontend/www/js/omegaup/components/problem/Settings.vue

```diff
@@ -178,5 +179,11 @@ export default class Settings extends Vue {
   onInitialLanguageChange(newInitial: string): void {
     this.languages = newInitial;
   }
+
+  @Emit('languages-changed')
+  onLanguagesChange(event: Event): string {
+    const newValue = (event.target as HTMLInputElement).value;
+    return newValue;
+  }
```

Comment:
```suggestion
  @Emit('update:languages')
  @Watch('currentLanguages')
  onLanguagesChange(newValue: string): string {
    return newValue;
  }
```

Vue prefiere usar eventos de la forma [`update:myFieldName`](https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier) para que sean compatibles con `v-model` y `.sync`.